src/components/userInfo/UserInfo.js

```diff
@@ -57,6 +72,16 @@ class UserInfo extends React.Component {
     };
   }
 
+  /*
+
+  ECS:
+  
+  Read up on promises for both methods below. getCurrentUserPicture feels overly complex with
+  chained .then() functions. It feels like handleDelete could use a .then() to only logout after the
+  deleteUser completes.
+  
```

Comment:
Yeah, it would've made more sense to .then the logout after the delete since of the async and needing to be authorized for delete user.